tfx/components/trainer/fn_args_utils.py

```diff
@@ -59,14 +59,14 @@ def get_common_fn_args(input_dict: Dict[Text, List[types.Artifact]],
                        working_dir: Text = None) -> FnArgs:
   """Get common args of training and tuning."""
   train_files = [
-      io_utils.all_files_pattern(
-          artifact_utils.get_split_uri(input_dict[constants.EXAMPLES_KEY],
-                                       'train'))
+      io_utils.all_files_pattern(uri) for uri in
+          artifact_utils.get_split_uris(input_dict[constants.EXAMPLES_KEY],
+                                        'train')
   ]
   eval_files = [
-      io_utils.all_files_pattern(
-          artifact_utils.get_split_uri(input_dict[constants.EXAMPLES_KEY],
-                                       'eval'))
+      io_utils.all_files_pattern(uri) for uri in
+          artifact_utils.get_split_uris(input_dict[constants.EXAMPLES_KEY],
+                                        'eval')
```

Comment:
can we have an executor_test case (new or update existing) to cover the code path

Reply:
added a test case, which specifically tests multiple artifact behavior